### ISSUE TITLE
console cache:clear command doesn't reload overrides

### ISSUE THREAD

**Issue by jf-viguier** [PrestaShop/PrestaShop#15610]
**Describe the bug**
When using commande line cache:clear, the class_index.php is not regenerated. 
The command should completly clear the cache like the clear cache button of the BO. 

**To Reproduce**
1. Add an override in override/ folder 
2. Clear the cache on command line : php -d memory_limit=-1 ./bin/console cache:clear
3. The override is not used
4. Clear the cache with the bo button
5. The override is used

**Additional information**
PrestaShop version: 1.7.6.1
PHP version: 7.0

**Comment by khouloudbelguith** [comment]
Hi @jf-viguier,

I manage to reproduce the issue with PS1.7.6.1.
This issue is blocked by this problem: https://github.com/PrestaShop/PrestaShop/issues/15324
Here's the PR: https://github.com/PrestaShop/PrestaShop/pull/15515 to fix the issue.
I tried with this PR & it is OK for me.
Your ticket is a duplicate of #15324
So, I close it.
Thanks!

**Comment by khouloudbelguith** [comment]
Duplicate of #15324
